<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-07-03T20:26:00 EEST" name="Suite1" finished-at="2024-07-03T20:26:31 EEST" duration-ms="30912">
    <groups>
    </groups>
    <test started-at="2024-07-03T20:26:00 EEST" name="test" finished-at="2024-07-03T20:26:31 EEST" duration-ms="30912">
      <class name="tests.ContactFormTest">
        <test-method is-config="true" signature="setup(java.lang.String,java.lang.String)[pri:0, instance:tests.ContactFormTest@27462a88]" started-at="2024-07-03T20:26:00 EEST" name="setup" finished-at="2024-07-03T20:26:05 EEST" duration-ms="4363" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[https://keyfood.ro]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="sendValidMessage()[pri:1, instance:tests.ContactFormTest@27462a88]" started-at="2024-07-03T20:26:05 EEST" name="sendValidMessage" finished-at="2024-07-03T20:26:17 EEST" duration-ms="11860" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sendValidMessage -->
        <test-method is-config="true" signature="recordFailure(org.testng.ITestResult)[pri:0, instance:tests.ContactFormTest@27462a88]" started-at="2024-07-03T20:26:17 EEST" name="recordFailure" finished-at="2024-07-03T20:26:17 EEST" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=sendValidMessage status=SUCCESS method=ContactFormTest.sendValidMessage()[pri:1, instance:tests.ContactFormTest@27462a88] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- recordFailure -->
        <test-method signature="sendInvalidMessage()[pri:2, instance:tests.ContactFormTest@27462a88]" started-at="2024-07-03T20:26:17 EEST" name="sendInvalidMessage" finished-at="2024-07-03T20:26:26 EEST" duration-ms="9186" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Thank you for your message. It has been sent.] but found [One or more fields have an error. Please check and try again.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Thank you for your message. It has been sent.] but found [One or more fields have an error. Please check and try again.]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at tests.ContactFormTest.sendInvalidMessage(ContactFormTest.java:49)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sendInvalidMessage -->
        <test-method is-config="true" signature="recordFailure(org.testng.ITestResult)[pri:0, instance:tests.ContactFormTest@27462a88]" started-at="2024-07-03T20:26:26 EEST" name="recordFailure" finished-at="2024-07-03T20:26:26 EEST" duration-ms="300" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=sendInvalidMessage status=FAILURE method=ContactFormTest.sendInvalidMessage()[pri:2, instance:tests.ContactFormTest@27462a88] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- recordFailure -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.ContactFormTest@27462a88]" started-at="2024-07-03T20:26:26 EEST" name="tearDown" finished-at="2024-07-03T20:26:31 EEST" duration-ms="5128" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- tests.ContactFormTest -->
    </test> <!-- test -->
  </suite> <!-- Suite1 -->
</testng-results>
